fxmanifest.lua

```diff
@@ -1,8 +1,9 @@
 fx_version 'cerulean'
+game 'gta5'
+
 name 'joehud'
 author 'joerogan#0001 / CosmoKramer'
 description 'HUD & Seatbelt & Speedlimiter System'
-game 'gta5'
```

Comment:
Is there any difference to moving this from below to above?

Reply:
No, I just thought that it looked a bit neater and is easier to identify which fxversion the script is as most other scripts use the layout that i did